webapp/.sass-lint.yml

```diff
@@ -0,0 +1,48 @@
+#########################
+## Sample Sass Lint File
+#########################
+# Linter Options
+options:
+  # Don't merge default rules
+  merge-default-rules: false
+  # Raise an error if more than 50 warnings are generated
+  max-warnings: 50
+  # Output file (lose loglevel from console output)
+  #output-file: 'buildlog/sass-linter.html'
+  #formatter: 'html'
+# Rule Configuration
+rules:
+  extends-before-mixins: 2
+  extends-before-declarations: 2
+  placeholder-in-extend: 2
+  mixins-before-declarations:
+    - 2
+    -
+      exclude:
+        - breakpoint
+        - mq
+
+  no-warn: 1
+  no-debug: 1
+  no-ids: 2
+  no-important: 2
+  hex-notation:
+    - 2
+    -
+      style: uppercase
+  indentation:
+    - 2
+    -
+      size: 2
+  property-sort-order:
+    - 1
+    -
+      order: 'alphabetical'
+  variable-for-property:
+    - 0
+    -
+      properties:
+        - margin
+      whitelist-values:
+        - auto
+        - '0'
```

Comment:
For consistence with eslint configs, use the .sasslintrc config file so we will only need to maintain one type of config file. [json2yml](https://www.json2yaml.com/) will save you a lot of pain.

Reply:
Done!